src/robusta/integrations/slack/sender.py

```
@@ -1,5 +1,7 @@
 import copy
+import json
 import logging
+import os
```

Ruff (0.8.2)

`os` imported but unused

Remove unused import: `os`

(F401)